.github/workflows/ci.yml

```diff
@@ -1,20 +1,21 @@
 name: CI
 
 on:
-  pull_request:
-    types: [review_requested]
+  push:
+    branches-ignore:
+      - master
```

Comment:
I’m confused; why not run everything on everything? Linting matters in master just as much as on PRs, and still could fail after two passing PRs are merged.